src/vault/script.rs

```diff
@@ -26,6 +26,10 @@ pub(crate) fn vault_trigger_withdrawal() -> ScriptBuf {
         // start with encoded leaf hash
         .push_opcode(OP_CAT) // encoded leaf hash
         .push_opcode(OP_CAT) // encoded leaf hash
+        .push_opcode(OP_SWAP) // bring input index to top of stack
+        .push_opcode(OP_DROP) // remove input index
```

Comment:
```suggestion
        // TODO: add an `index` field to the CommitmentSpec so that we can exclude it and remove the OP_SWAP and OP_DROP here
        .push_opcode(OP_DROP) // remove input index
```

Reply:
just add this little TODO so in the future we can remove these two opcodes and save a little witness space :)